sdks/python/apache_beam/transforms/util.py

```diff
@@ -785,7 +788,6 @@ def expand(self, pcoll):
 
   @experimental()
   @typehints.with_input_types(Tuple[K, V])
-  @typehints.with_output_types(Tuple[K, Iterable[V]])
```

Comment:
Just curious, does `@typehints.with_output_types` not work here?

Reply:
Udi also had a similar question. I got TypeError when trying to use Tuple[ShardedKey,...]. See my reply below.